### PR TITLE
Replay button now will be shown only for the last video in playlist

### PR DESCRIPTION
**Pull request by RomanTysiachnik**
@VerizonAdPlatforms/mobile-sdk-developers: Please review.

### CODE REVIEW

File: PlayerControls/sources/ContentControlsUIProps.swift [hunk 1]
```diff
@@ -124,7 +124,10 @@ extension DefaultControlsViewController {
             
             pauseButtonAction = props.player?.item.playable?.playbackAction.pause ?? .nop
             
-            replayButtonHidden = props.player?.item.playable?.playbackAction.replay == nil
+            replayButtonHidden = {
+                guard props.player?.item.playable?.playbackAction.replay != nil else { return true }
```
Comment (AndriiMoskvin) [inline PlayerControls/sources/ContentControlsUIProps.swift]: This looks better:
```
replayButtonHidden = {
                guard let player = props.player else { return true }
                guard player.item.playable?.playbackAction.replay != nil else { return true }
                guard let playlist = player.playlist else { return false }
                return !playlist.isLastVideoInPlaylist
            }()
```

Comment (RomanTysiachnik) [reply PlayerControls/sources/ContentControlsUIProps.swift]: it is done

File: PlayerControls/sources/ContentControlsViewController.swift [hunk 2]
```diff
@@ -240,7 +240,7 @@ extension ContentControlsViewController.Props.Player {
 }
 
 extension ContentControlsViewController.Props.Playlist {
-    public init(next: Command?, prev: Command?) {
+    public init(next: Command?, prev: Command?, isLastVideoInPlaylist: Bool) {
```
Comment (AndriiMoskvin) [inline PlayerControls/sources/ContentControlsViewController.swift]: This can be dropped.

Comment (RomanTysiachnik) [reply PlayerControls/sources/ContentControlsViewController.swift]: Done

**Pull request merged**